Major naming refactoring (#15)
Caliatys/CognitoService: README.md

  constructor(public cognitoHelper : CognitoHelper)
  {
    // this.cognitoHelper.cognitoService...

> 

Reviewer: ```suggestion

```

Author: Done! 😄

## Variables

> Events that you can subscribe to deal with login and logout (signOut) events

Reviewer: Apply the naming convention
```suggestion
Events that you can subscribe to deal with `signIn` and `signOut` events
```

Author: Done! 😄

```typescript
// Events that you can subscribe to

> public onLogin  : EventEmitter<null>;

Reviewer: Please update the two events with the naming convention (`onSignIn`, `onSignOut`).
It must be changed everywhere : sample, service & doc.
```suggestion
public onSignIn  : EventEmitter<null>;
```

#### Resend confirmation code
```typescript
this.cognitoHelper.cognitoService.resendConfirmationCode();
```

#### Login
Login an existing user with Google or Cognito.

##### Google
```typescript

> this.cognitoHelper.cognitoService.authenticateUser(AuthType.GOOGLE).subscribe(res =>

Reviewer: Apply the naming convention :
```suggestion
this.cognitoHelper.cognitoService.signIn(AuthType.GOOGLE).subscribe(res =>
```

Author: Done! 😄

##### Cognito
```typescript

> this.cognitoHelper.cognitoService.authenticateUser(AuthType.COGNITO, 'username', 'password').subscribe(res => {

Reviewer: Apply the naming convention :
```suggestion
this.cognitoHelper.cognitoService.signIn(AuthType.COGNITO, 'username', 'password').subscribe(res => {
```

Author: Done! 😄
